ratis-server/src/main/java/org/apache/ratis/server/impl/FollowerInfoImpl.java

```diff
@@ -202,4 +204,15 @@ public Timestamp getLastRpcTime() {
   public Timestamp getLastHeartbeatSendTime() {
     return lastHeartbeatSendTime.get();
   }
+
+  @Override
+  public Timestamp getLastRespondedAppendEntriesSendTime() {
+    return lastRespondedAppendEntriesSendTime.get();
+  }
+
+  @Override
+  public void updateLastAppendEntriesResponseTime(Timestamp sendTime) {
+    updateLastRpcResponseTime();
+    lastRespondedAppendEntriesSendTime.set(sendTime);
+  }
```

Comment:
Let's call it` updateLastRespondedAppendEntriesSendTime` and update only the sendTime. It will be easier to understand. i.e.
```java
  @Override
  public void updateLastRespondedAppendEntriesSendTime(Timestamp sendTime) {
    lastRespondedAppendEntriesSendTime.set(sendTime);
  }
```

Reply:
Sure.